src/DefaultBuilder/test/Microsoft.AspNetCore.Tests/WebApplicationTests.cs

```diff
@@ -364,101 +373,135 @@ public void WebApplicationBuilderSettingInvalidApplicationWillFailAssemblyLoadFo
         [Fact]
         public void WebApplicationBuilderCanConfigureHostSettingsUsingWebApplicationOptions()
         {
-            var contentRoot = Path.GetTempPath().ToString();
-            var webRoot = Path.GetTempPath().ToString();
+            var contentRoot = Path.GetTempPath();
+            var webRoot = "wwwroot";
+            var fullWebRootPath = Path.Combine(contentRoot, webRoot);
+            Directory.CreateDirectory(fullWebRootPath);
```

Comment:
Nit: Since we're improving this, can we create a new random temp directory for the content root? In theory, if these tests were run in parallel, another test could delete the web root directory while this is still running.

Also, doesn't the web root directory get created automatically?

Reply:
Yep, the WebRoot directory will be created automatically but we'll need to create/remove the unique content root directories.